lib/tasks/debug.rake

```diff
@@ -78,4 +79,9 @@ namespace :debug do
       end
     end
   end
+
+  desc "Run the core of the search query with explain plans enabled and special boosting disabled"
+  task :explain, [:query] do |_, args|
+    Debug::Explainotron.explain!(args.query)
```

Comment:
Should this pass in the A/B test parameter? (I realise this is a WIP, so you might be planning to add it later.)

Reply:
Possibly... I'm not sure what state I want to leave this in yet.